src/styles/navigation.scss

```diff
@@ -0,0 +1,83 @@
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 $rhythm;
+  font-family: $primary-font;
+  height: $rhythm * 6;
+  border-bottom: 1px solid var(--border-primary);
+
+  .nav-icon {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    background-color: transparent;
+    border: none;
+    outline: none;
+    padding: 0;
+
+    h1 {
+      margin-left: $rhythm !important;
+      font-size: $rhythm * 2;
+      font-weight: bold;
+      color: var(--medium-grey);
+    }
```

Comment:
similar to the above mentioned flex mixin you can create one for text too.

Reply:
#14 Issue has been created and PR will be sent soon

Reply:
COOOOOOOOOOOOOOL